src/coreclr/gc/unix/numasupport.cpp

```diff
@@ -54,7 +54,7 @@ static int GetNodeNum(const char* path, bool firstOnly)
 void NUMASupportInitialize()
 {
 #ifdef TARGET_LINUX
-    if (syscall(__NR_get_mempolicy, NULL, NULL, 0, 0, 0) < 0 && errno == ENOSYS)
+    if (syscall(__NR_get_mempolicy, NULL, NULL, 0, 0, 0) < 0 && (errno == ENOSYS || errno == EPERM))
```

Comment:
@janvorli, an alternative was to drop && errno filter and just return for `< 0`. What do you think?
```suggestion
    if (syscall(__NR_get_mempolicy, NULL, NULL, 0, 0, 0) < 0)
```

Reply:
I am fine with both ways, but maybe I'd slightly prefer not checking the errno at all as it would be more future proof.